runtime/doc/eval.txt

```diff
@@ -4891,7 +4892,7 @@ readfile({fname} [, {binary} [, {max}]])
 		separated with CR will result in a single long line (unless a
 		NL appears somewhere).
 		All NUL characters are replaced with a NL character.
-		When {binary} is equal to "b" binary mode is used:
+		When {binary/append} contains "b" binary mode is used:
```

Comment:
You're right, this is confusing.. I guess the `{binary}` parameter was supposed to be renamed to `{binary/append}`? It's still unchanged in current Vim..